### ISSUE TITLE
DropDownMenu not worked

### ISSUE THREAD

**Issue by iwahara** [JetBrains/compose-jb#100]
when DropDownMenu click, throw `NoSuchMethodError`.
I am using version `0.1.0-m1-build62`.

```
Caused by: java.lang.NoSuchMethodError: 'void androidx.compose.ui.window.PopupKt.Popup(androidx.compose.ui.window.PopupPositionProvider, boolean, kotlin.jvm.functions.Function0, androidx.compose.ui.window.PopupProperties, kotlin.jvm.functions.Function2, androidx.compose.runtime.Composer, int, int)'
```

Reproduction code:

```kotlin
import androidx.compose.desktop.Window
import androidx.compose.foundation.clickable
import androidx.compose.foundation.layout.fillMaxWidth
import androidx.compose.material.DropdownMenu
import androidx.compose.material.DropdownMenuItem
import androidx.compose.material.MaterialTheme
import androidx.compose.material.Text
import androidx.compose.runtime.Composable
import androidx.compose.runtime.mutableStateOf
import androidx.compose.runtime.remember
import androidx.compose.ui.Modifier
import androidx.compose.ui.unit.IntSize

fun main() = Window(title = "Compose for Desktop", size = IntSize(600, 600)) {
    MaterialTheme {
        DropdownDemo()
    }
}

@Composable
fun DropdownDemo() {
    val items = listOf("A", "B", "C", "D", "E", "F")
    val showMenu = remember { mutableStateOf(false) }
    val selectedIndex = remember { mutableStateOf(0) }

    DropdownMenu(
            toggle = {
                Text(items[selectedIndex.value], modifier = Modifier.fillMaxWidth().clickable(onClick = { showMenu.value = true }))
            },
            expanded = showMenu.value,
            onDismissRequest = { showMenu.value = false },
    ) {
        items.forEachIndexed { index, s ->
            DropdownMenuItem(
                    onClick = {
                        selectedIndex.value = index
                        showMenu.value = false
                    }
            ) {
                Text(text = s)
            }
        }
    }
}

```

What should I do?

**Comment by Cybermaxke** [comment]
Seems to be working with `0.2.0-build127`.

**Comment by iwahara** [comment]
I made sure this issue is fixed in `0.2.0-build128`.
thanks for telling me.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6512">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
